src/current/v23.2/node-shutdown.md

```diff
@@ -76,31 +89,35 @@ Node drain consists of the following consecutive phases:
 1. **Lease transfer phase:** The node's [`is_draining`]({% link {{ page.version.version }}/cockroach-node.md %}#node-status) field is set to `true`, which removes the node as a candidate for replica rebalancing, lease transfers, and query planning. Any [range leases]({% link {{ page.version.version }}/architecture/replication-layer.md %}#leases) or [Raft leaderships]({% link {{ page.version.version }}/architecture/replication-layer.md %}#raft) must be transferred to other nodes. This phase completes when all range leases and Raft leaderships have been transferred.
 
     <section class="filter-content" markdown="1" data-scope="decommission">
-    Since all range replicas were already removed from the node during the [decommissioning](#decommissioning) stage, this step immediately resolves.
+    Since all range replicas were already removed from the node during the [draining](#draining) stage, this step immediately resolves.
```

Comment:
```suggestion
    Since all [range replicas]({% link {{ page.version.version }}/architecture/glossary.md %}#replica) were already removed from the node during the [draining](#draining) stage, this step immediately resolves.
```